app/Http/Controllers/Api/V1/Admin/ProjectController.php

```diff
@@ -42,7 +42,7 @@ public function __construct(ProjectRepository $projectRepository)
      */
     public function index(Request $request)
     {
-        return ProjectResource::collection($this->projectRepository->getAll());
+        return ProjectResource::collection($this->projectRepository->getAll($request->all()));
```

Comment:
What is this?

Reply:
Passing DataTables params to the repository. Could be order, length, search etc...

Reply:
If it has other params, how does it work? Throwing error?

Reply:
Other params will be neglected. No exception or error will be thrown.

Reply:
If there is some other action needs to be performed on the additional params that can be achieved very easily. So this will work for all use cases not only for data-tables.